src/data/compositors/weston.json

```diff
@@ -1,106 +1,94 @@
 {
-  "generationTimestamp": 1709750464116,
-  "version": "13",
+  "generationTimestamp": 1737939500990,
+  "version": "0.14",
```

Comment:
The previous version was correct, it's `14` not `0.14` (Weston 1.0.0 was released ~12 years ago :wink: )